### ISSUE TITLE
How to import PoolOption?

### ISSUE THREAD

**Issue by t4sk** [Couragium/rsmq-async-rs#11]
How do I import `PoolOption` needed to create a pooled connection?

[pool_facade.rs](https://github.com/Couragium/rsmq-async-rs/blob/master/src/pooled_facade.rs#L84)
```rust
    pub async fn new(options: RsmqOptions, pool_options: PoolOptions) -> RsmqResult<PooledRsmq> {
```

It's not exported from [lib.rs](https://github.com/Couragium/rsmq-async-rs/blob/master/src/lib.rs#L158-L166)

**Comment by DavidBM** [comment]
Hi! I have added a commit exporting it. Later today I will publish a new version in crates.io.
Use this report ufor now. Tomorrow it should be solved

**Comment by t4sk** [comment]
Wow quick response! Thanks

**Comment by DavidBM** [comment]
I had some problems during my flight, so I wasn't able to upload the package to crates.io yet. Will try today! I will reopen this as a reminder :)

**Comment by DavidBM** [comment]
Ok! My flight had wifi, so I just published the new version of the crate! 5.1.0 @t4sk

**Comment by t4sk** [comment]
Thanks
